index.html

```diff
@@ -0,0 +1,1710 @@
+                    </svg>
+                  </a> -->
+              </div>
+
+              <div class="col-sm-6 col-md-4 col-lg-3  d-flex flex-lg-row
+              justify-content-end font-weight-bold d-font">
+
+                  <a class="d-sm-inline-block my-sm-2 text-white text-uppercase
+                  text-decoration-none pt-sm-1 mr-lg-2 sign-in" href="#">Sign
+                  In</a>
+
+                  <a class="d-sm-inline-block mt-sm-1 mx-sm-1 px-sm-1 text-danger bg-white
+                  text-uppercase  text-decoration-none " href="#">
+                  <p class="py-2 px-sm-2 px-lg-2 d-lg-inline-block subscribe"> Subscribe</p> <i class="fa
+                  fa-angle-right d-sm-none d-lg-inline-block arrw" aria-hidden="true"></i></a>
+
+              </div>
+
+        </div>
+      </nav>
+
+      <nav class="py-lg-4 nav-bottom d-sm-none d-md-none d-lg-block ">
+          <ul class="navbar-nav big-nav d-flex flex-lg-row  ">
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#" >U.S.</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">World</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Business</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Tech & Science</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Culture</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Sports</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Health</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Opinion</a>
+              </li>
+              <li class="nav-li nav-item">
+                <a class="nav-link" href="#">Ndelible</a>
+              </li>
+          </ul>
+      </nav>
+  </header>
+
+<!-- MAIN SECTION -->
+<div class="container-fluid">
```

Comment:
It is of a good practice to make use of `main tag` on this line, checkout documentation for that [here](https://www.themelocation.com/best-html5-practices/)